packages/p2p/src/components/orders/order-table/order-table.jsx

```diff
@@ -27,6 +27,19 @@ const OrderTable = ({ showDetails }) => {
     const is_active_tab = general_store.order_table_type === order_list.ACTIVE;
     return (
         <React.Fragment>
+            {general_store.banner_config.reduced_order_time && (
+                <div className='orders__banner'>
+                    <HintBox
+                        icon='IcInfo'
+                        message={
+                            <Text as='p' size='xxxs' color='prominent' line_height='xs'>
+                                <Localize i18n_default_text='New orders are now active for 1 hour only. Complete your order before it expires!' />
+                            </Text>
+                        }
+                        is_info
+                    />
```

Comment:
You're reusing the same component twice, why not create a separate component for it?

Reply:
If this is a temporary feature, there is no need for separate component